# COVID-19 at WikiPathways
Here is a listing of the pathways being modeled and curated by the community at WikiPathways. The collection can be viewed at this dedicated portal page: 
* https://www.wikipathways.org/index.php/Portal:Disease/COVIDPathways

And all pathways tagged for our Current Highlight (which is COVID-19, of course) can be downloaded in bulk in multiple formats:
* [GPML](https://www.wikipathways.org//wpi/batchDownload.php?species=Homo%20sapiens&fileType=gpml&tag=Curation:CurrentHighlight)
* [SVG](https://www.wikipathways.org//wpi/batchDownload.php?species=Homo%20sapiens&fileType=svg&tag=Curation:CurrentHighlight)
* [PNG](https://www.wikipathways.org//wpi/batchDownload.php?species=Homo%20sapiens&fileType=png&tag=Curation:CurrentHighlight)

We will also try to keep the listing in this document up-to-date as well

## SARS-CoV-2 and COVID-19 Pathway
https://www.wikipathways.org/index.php/Pathway:WP4846
Collaborative project for curation biological processes involved in the COVID-19 disease after SARS-Cov-2 infection. Editing this pathway is (at this moment) coordinated via the wikipathways.slack.com #sarscov2 channel.
The large viral Spike (S) protein (surface glycoprotein) needs to be primed by host proteases. For SARS-CoV-2 that is the serine protease TMPRSS2. After priming the virus uses the host's ACE2 receptor for entry (Hoffmann et al 2020). The ACE2 receptor interaction was also suggested as the start of specfic lung damaging effects 

## https://www.wikipathways.org/index.php/Pathway:WP4853
https://www.wikipathways.org/index.php/Pathway:WP4853
Lipid metabolism alternations that are related to infection by corona viruses. The information comes from the Yan et al. in 2019 in the bibliography, particularly Figure 5. That paper uses the HCoV-229E virus as a model. Note that that is different from the virus that causes the 2020 pandemic SARS-CoV-2. Fig 5 is in turn taken from https://www.kegg.jp/kegg-bin/show_pathway?map00591, which is a very simplified pathway, omitting several steps The paper mentions that after virus infection many of the metabolites in his figure are increased in concentration. Interestingly, exogenous supplement of LA or AA in HCoV-229E-infected cells significantly suppressed HCoV-229E virus replication and this also happened in MERS-CoV.

## Hijack of Ubiquitination by SARS-CoV-2
https://www.wikipathways.org/index.php/Pathway:WP4860
SARS-CoV-2 includes a novel Orf10 that interacts with muliple members of the Cullin 2 ubiquitin ligase complex as determined by AP-MS (Gordon 2020). The strongest interaction is with ZYG11B, a substrate adaptor for CUL2. By binding this complex, Orf10 might be able to hijack its activity. The hijacking of ubiquitination machinery is a common strategy of viruses to direct the degradation of viral restriction factors, for example. Also depicted here is the required neddylation (N8) of CUL2 by the NAE enzyme complex. The ability of this enzyme to transfer N8 to CUL2 is inhibited by the small molecule Pevonedistat.

## Type I Interferon Induction and Signaling During SARS-CoV-2
https://www.wikipathways.org/index.php/Pathway:WP4868
The induction of Type I interferons and signaling leading to the innate immune response during SARS-COV-2 infection. TLR7 MYD88-dependent signaling is inhibited at multiple steps by the SARS-CoV Papain-Like Protease (PLpro) domain of nsp3 (red oval). The signaling pathway is critical to induction of type I interferons (INF-I) via IRF3, AP-1 and NFkB transcription factors. INF-I triggers the JAK/STAT pathway leading to the induction of interferon-stimulated genes (ISGs), such as OAS and PKR, which go one to conduct the innate immune response. TREML4 has been shown to be necessary for MYD88 recruitment by TLR7 and STAT1 participation. The inhibition of SARS-CoV-2 PLpro by GRL0617 is proposed based on Ratia, et al. 2008 and 100% sequence identity between SARS-CoV and SARS-CoV-2 across all 13 residues of PLpro involved in binding GRL0617 (82.9% identity across 316 amino acids) as determined by the alignment of RefSeq YP_009725299.1 and PDB 3E9S.

## Host-pathogen interaction of human corona viruses - autophagy
https://www.wikipathways.org/index.php/Pathway:WP4863
Induction and modulation of autophagy system during human corona virus infection.

## Host-pathogen interaction of human corona viruses - apoptosis
https://www.wikipathways.org/index.php/Pathway:WP4864
Apoptosis induction by human corona virus infection.

## Host-pathogen interaction of human corona viruses - ER stress
https://www.wikipathways.org/index.php/Pathway:WP4869
ER stress due to unfolded protein response. Figure 6 of Fung and Liu 2019. 

## Downregulation of ACE2 expression by SARS-CoV infection and SARS-CoV Spike protein.
https://www.wikipathways.org/index.php/Pathway:WP4799
Schematic diagram of the renin-angiotensin system in acute lung failure and proposed SARS-CoV action. The renin-angiotensin system has a crucial role in severe acute lung injury and the SARS-CoV receptor ACE2 has a protective role in acute lung failure. Because ACE2 is a crucial SARS-CoV receptor, SARS-CoV Spike protein binding to ACE2 downmodulates ACE2 expression, and loss of ACE2 expression results in severe acute respiratory failure. The AT1R is the crucial receptor that mediates AngII-induced vascular permeability and severe acute lung injury. SARS-CoV Spikeâ€“mediated lung failure can be rescued by inhibition of AT1R. This research was done in in vivo mice cells.
